src/calculus.jl

```diff
@@ -0,0 +1,10 @@
+
+
+function diff(b1::Basic, b2::Basic)
```

Comment:
Is there a means to get the contents of SYMENGINE_ENUM as a dictionary in
julia?

Something like

```
const SYMENGINE_ENUM = Dict{Int, Symbol}(0 => :Integer,
                                         1 => :Rational,
                                         2 => :Complex,
                                         3 => :ComplexDouble,
                                         11 => :Symbol,
                                         12 => :EmptySet,
                                         13 => :Interval,
                                         14 => :Mul,
                                         15 => :Add,
                                         16 => :Pow)
```

where the numbers are guesses and seem likely to change?


On Fri, Mar 25, 2016 at 3:55 PM, Isuru Fernando <notifications@github.com>
wrote:

> In src/calculus.jl
> <https://github.com/symengine/SymEngine.jl/pull/17#discussion_r57483734>:
>
> > @@ -0,0 +1,10 @@
> > +
> > +
> > +function diff(b1::Basic, b2::Basic)
>
> Yes, basic_get_type is the function to be used.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/symengine/SymEngine.jl/pull/17/files/7343bd913def49c83b58a274d6363243ba1b9b57#r57483734>
>



-- 
John Verzani
Chair, Department of Mathematics
College of Staten Island, CUNY
verzani@math.csi.cuny.edu